.travis_build.sh

```diff
@@ -105,7 +105,7 @@ case "$PLATFORM" in
     pushd "${build_directory}"
 
     travis_fold start build_vm "Building OpenSmalltalk VM..."
-    ./mvm -f
+    make
```

Comment:
I don't know the answer, but i wonder, is it still necessary?
From my understanding, there's been 2 changes in this area:
- https://github.com/OpenSmalltalk/opensmalltalk-vm/commit/46d8e8cd25d9b92ffd21ce21dcdeb2406b18c804
- https://github.com/OpenSmalltalk/opensmalltalk-vm/commit/3e2b3e429307bff51036ebdf551261e715af337b
the LOGD artifacts might be of importance (we could track non regresion of warnings for example) and it won't be produced by direct call to make.